apisix/plugins/schema.lua

```diff
@@ -0,0 +1,18 @@
+local typedefs = require "kong.db.schema.typedefs"
```

Comment:
Why this PR contains Kong's schema?

Reply:
Cause this was generated by ChatGPT

Reply:
@Neilblaze Sorry buddy, but it seems like you're just jealous. But don't worry, I'll continue to thrive and you can continue to wallow in your own insecurities. 😎 and Thanks for reminding me of this PR.

Reply:
> Why this PR contains Kong's schema?

@spacewander By using Kong's schema, the OIDC plugin is able to ensure that the claim_validators field is properly formatted and validated before it is used.

Reply:
@ro4i7 No need to be sorry. People often rant when they get caught 🤣
Talk is cheap, show me the code.